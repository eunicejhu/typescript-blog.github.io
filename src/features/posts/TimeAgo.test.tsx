import React from "react";
import { render } from "@testing-library/react";
import TimeAgo from "./TimeAgo";
import getTimeAgo from "../../utils/getTimeAgo";

jest.mock("../../utils/getTimeAgo.ts");
test("render correctly", () => {
  (getTimeAgo as jest.Mock).mockReturnValue("28 minutes");
  const date = "2020-11-15T16:16:08.493Z";
  const ui = <TimeAgo date={date}></TimeAgo>;
  const { asFragment } = render(ui);
  expect(asFragment()).toMatchSnapshot();
});
