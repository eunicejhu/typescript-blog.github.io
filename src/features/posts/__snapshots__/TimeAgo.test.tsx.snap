// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<DocumentFragment>
  <span
    class="date"
  >
     
    <i>
      28 minutes ago
    </i>
     
  </span>
</DocumentFragment>
`;
